cassdegrees/templates/managecourses.html

```diff
@@ -0,0 +1,115 @@
+{% extends "base.html" %}
+
+{% block page-title %}Manage Courses{% endblock %}
+
+{% block subtitle %}Add, Modify or Delete Courses{% endblock %}
+
+{% block content %}
+    <form class="anuform" action="/manage_courses/" method="post">
+        {% csrf_token %}
+        <fieldset>
+            <legend>Add Course</legend>
+
+            <!-- As HTML forms only support get or post methods,
+            add this hidden input for consistency with the rest of the forms -->
+            <input type="hidden" name="_method" value="post">
+
+            <p>
+                <label for="id">Course Name:</label>
+                <input class="text" id="text" type="text" name="name" aria-required="true" required>
```

Comment:
```suggestion
                <input class="text tfull" id="text" type="text" name="name" aria-required="true" required>
```

Makes the text box stretch to the end of the page, giving the user more space to type

Reply:
good idea